src/olympia/abuse/admin.py

```diff
@@ -353,18 +353,33 @@ def message_excerpt(self, obj):
 
 class CinderPolicyAdmin(AMOModelAdmin):
     fields = (
+        'id',
         'created',
         'uuid',
+        'parent',
         'name',
         'text',
     )
     list_display = (
+        'id',
         'uuid',
+        'parent',
         'name',
+        'text',
```

Comment:
might be a become a bit busy with the text, but 🤷

Reply:
It will, but no worse than in Cinder, and I think it's helpful to distinguish between all the noise